### ISSUE TITLE
Remove avatar when deleting an account/moving WCA ID/anonymizing account

### ISSUE THREAD

**Issue by dunkOnIT** [thewca/worldcubeassociation.org#8707]
Currentyl when trying to transfer a WCA ID or delete an account, the user is informed that they have to first remove the avatar. This is annoying for the informed user and confusing for the uninformed user.

We should instead flash a warning saying the the avatar will be deleted, requesting confirmation. 

I think this is better than silently deleting it, as it may not be obvious that deleting the WCA ID (not the user account) would cause the avatar to be removed).

**Comment by dunkOnIT** [comment]
I don't think this is a first issue as a lot of the avatar handling code look confusing (to me at least)

**Comment by Nanush7** [comment]
A classic JS alert would be enough? Or you are moving to something else?
